controllers/api.js

```diff
@@ -0,0 +1,2 @@
+const endpoints = require("../endpoints.json")
```

Comment:
I took this exact approach when doing this project, and got called out for sending the whole file, therefore I suggest that you export the object from the file and send that instead